src/neo/IO/Helper.cs

```diff
@@ -162,9 +162,30 @@ public static byte[] ReadBytesWithGrouping(this BinaryReader reader)
             }
         }
 
+        public static byte[] ReadFixedBytes(this BinaryReader reader, int size)
+        {
+            var index = 0;
+            var data = new byte[size];
+
+            while (size > 0)
+            {
+                var readed = reader.Read(data, index, size);
+
+                if (readed <= 0 && size > 0)
```

Comment:
how can size ever be <= 0 here? it should be impossible, since the while condition will be false if size was <= 0.